doc/releases.md

```diff
@@ -574,10 +574,9 @@ computer.
 by GPG for your password. The signed file will be named SHASUMS256.txt.asc.
 
 **f.** Output an ASCII armored version of your public GPG key using a command
-similar to: `gpg --default-key YOURKEY --armor --export --output
-/path/to/SHASUMS256.txt.gpg`. This does not require your password and is mainly
-a convenience for users, although not the recommended way to get a copy of your
-key.
+similar to: `gpg --default-key YOURKEY --detach-sign /path/to/SHASUMS256.txt`.
```

Comment:
we have `--digest-algo SHA256` explict in release.sh so it might be best to include that here too

Reply:
fixed